blog/_posts/2021-06-17-uncertainty.md

```diff
@@ -0,0 +1,244 @@
+estimate confidence [^9]. They do this by learning TCP with a second network trained to predict the softmax probability of 
+the groundtruth class generated by a pretrained and frozen classification network.  
+
+![](/blog/images/uncertainty/confidencenet.jpg)
+
+
+
+
+
+
+# Further reading
+
+## Review articles 
+The following articles offer reviews of many different methods. 
+
+* *Assessing Reliability and Challenges of Uncertainty Estimations for Medical Image Segmentation*, Jungo et al. 2019.
+* *Estimating uncertainty in deep learning for reporting confidence to clinicians in medical image segmentation and diseases detection*, Ghoshal et al. 2020.
+* *Leveraging Uncertainty Estimates for Predicting Segmentation Quality*, DeVries et al. 2018
+
+## Other 
+
+* *Local Temperature Scaling for Probability Calibration*, Ding et al. 2020 (Applying temperature scaling to segmentation)
+
+# References
+
+[^1]: Blundell, C, Cornebise, J, Kavukcuoglu, K, and Wierstra, D. Weight uncertainty in neural networks. ICML, 2015.
+
+[^2]: Yarin Gal and Zoubin Ghahramani. Dropout as a Bayesian approximation: Insights and applications. In Deep Learning Workshop, ICML, 2015.
+
+[^3]: Regression with Probabilistic Layers in TensorFlow Probability. Pavel Sountsov, Chris Suter, Jacob Burnim, Joshua V. Dillon, and the TensorFlow Probability team. March 12, 2019.  https://blog.tensorflow.org/2019/03/regression-with-probabilistic-layers-in.html
+
+[^4]: Guo, C., Pleiss, G., Sun, Y. and Weinberger, K.Q. On Calibration of Modern Neural Networks. In ICML, 2017.
+
+[^5]: Niculescu-Mizil, Alexandru and Caruana, Rich. Predicting good probabilities with supervised learning. In ICML, pp. 625–632, 2005.
+
+[^6]: Naeini, Mahdi Pakdaman, Cooper, Gregory F, and Hauskrecht, Milos. Obtaining well calibrated probabilities using bayesian binning. In AAAI, pp. 2901, 2015.
+
+[^7]: Kendall, A., & Gal, Y. (2017). What Uncertainties Do We Need in Bayesian Deep Learning for Computer Vision? In I. Guyon, U. V. Luxburg, S. Bengio, H. Wallach, R. Fergus, S. Vishwanathan, & R. Garnett (Eds.), Advances in Neural Information Processing Systems (Vol. 30). Curran Associates, Inc. https://proceedings.neurips.cc/paper/2017/file/2650d6089a6d640c5e85b2b88265dc2b-Paper.pdf
+
+[^8]: Balaji Lakshminarayanan, Alexander Pritzel, and Charles Blundell. 2017. Simple and scalable predictive uncertainty estimation using deep ensembles. In Proceedings of the 31st International Conference on Neural Information Processing Systems (NIPS'17). Curran Associates Inc., Red Hook, NY, USA, 6405–6416.
+
+[^9]: Corbière, C., THOME, N., Bar-Hen, A., Cord, M., & Pérez, P. (2019). Addressing Failure Prediction by Learning Model Confidence. In H. Wallach, H. Larochelle, A. Beygelzimer, F. d\textquotesingleAlché-Buc, E. Fox, & R. Garnett (Eds.), Advances in Neural Information Processing Systems (Vol. 32). Curran Associates, Inc. https://proceedings.neurips.cc/paper/2019/file/757f843a169cc678064d9530d12a1881-Paper.pdf
```

Comment:
\textquotesingle: probably slipped from the bibtex file.

Reply:
Fixed in eacdc9f

Reply:
Done in eacdc9f.